### ISSUE TITLE
How to use your pretrained model for question-answering ? # Question

### ISSUE THREAD

**Issue by allaabdella2** [zihangdai/xlnet#261]
Hello, 
I'm new to NLP, I run the colab jupyter notebook you provided. But, I couldn't find the instructions on how I can use your model for Q&A analysis.

For example:
I have text, I want to ask a question and get some answers by using your model. Can you provide an example on this case. 

Thank you!

**Comment by luantran1311** [comment]
See if this can help https://huggingface.co/transformers/model_doc/xlnet.html#tfxlnetforquestionansweringsimple

**Comment by Sankalp1233** [comment]
This does help a little bit: https://huggingface.co/transformers/model_doc/xlnet.html#xlnetforquestionanswering but at the end of the code we have: outputs = model(input_ids, start_positions=start_positions, end_positions=end_positions)  loss = outputs.loss how do we get the answer to the question I am a little bit confused. They have the question but they do not show the answer specifically.
